src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,254 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static Calculator calculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void calculate_additionTwoPositiveIntegers_ok() {
+        double expected = 30;
+        double actual = calculator.calculate(10, 20, '+');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void calculate_additionTwoNegativeIntegers_ok() {
+        double expected = -30;
+        double actual = calculator.calculate(-10, -20, '+');
+        assertEquals(expected, actual);
```

Comment:
```suggestion
        assertEquals(expected, actual, delta);
```
In some cases, you'll have to use delta value in assertEquals for doubles. Difference should be less than DELTA in order for test to pass;
Это из условий задания и не забудь сделать ее constant. Я кстати так и не нашел в каких случаях ее применять поэтому для надежности использовал везде